fix_format.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2020, Arm Limited, All Rights Reserved
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#          http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+find . -name \*.md | xargs -L 1 vmdfmt -cols 100 -w -l
```

Comment:
You can execute `vmdfmt` on a directory and it will recursively execute the comment on any of its files. But we should probably write that in the `README.md` as well 😄 
The command: `vmdfmt -cols 100 -w -l src/` would do it

Reply:
Yeah, I'll add that in the README. I'll actually suggest `./`, not `src/` - in case you update README.md for example.

Reply:
I did only target `src` so that `CONTRIBUTING.md` and `README.md` do not have to be formatted with Versioned Markdown Format and we can have a bit more liberty with those.
If we decide this we would have to fix those files to be under VMD format (they are not currently because of the HTML comments) and also modify the CI script to check for those as well:
```diff
-MARKDOWN_FILES=`find src -name "*.md"`
+MARKDOWN_FILES=`find . -name "*.md"`
```
and probably in the future make all of our Markdown files in other projects consistent with that as well 🙄